sdk/securityinsights/Azure.ResourceManager.SecurityInsights/src/Customization/EntityData.cs

```diff
@@ -8,14 +8,16 @@
 using Azure.Core;
```

Comment:
any reason we have to move this to customized code?

Reply:
@fengzhou-msft had some comment on this one: https://github.com/Azure/azure-sdk-for-net/pull/32480#discussion_r1031018244